recipe/activate.sh

```diff
@@ -7,33 +7,60 @@
 # doesn't come with a deactivate script, because the symlink
 # is benign and doesn't need to be deleted.
 
+_la_log() {
+    if [ "$CF_LIBARROW_ACTIVATE_LOGGING" = "1" ]; then
+        echo "${BASH_SOURCE[0]} DEBUG: $*"
+    fi
+}
+
+_la_log "Beginning libarrow activation."
+
 # where the GDB wrappers get installed
-GDB_PREFIX="$CONDA_PREFIX/share/gdb/auto-load"
+_la_gdb_prefix="$CONDA_PREFIX/share/gdb/auto-load"
 
 # this needs to be in sync with ARROW_GDB_INSTALL_DIR in build.sh
-PLACEHOLDER="replace_this_section_with_absolute_slashed_path_to_CONDA_PREFIX"
+_la_placeholder="replace_this_section_with_absolute_slashed_path_to_CONDA_PREFIX"
 # the paths here are intentionally stacked, see #935, resp.
 # https://github.com/apache/arrow/blob/master/docs/source/cpp/gdb.rst#manual-loading
-WRAPPER_DIR="$GDB_PREFIX/$CONDA_PREFIX/lib"
+_la_wrapper_dir="$_la_gdb_prefix/$CONDA_PREFIX/lib"
 
-mkdir -p "$WRAPPER_DIR" || true
-# If the directory is not writable, nothing can be done
-if [ ! -w "$WRAPPER_DIR" ]; then
-    return
-fi
+_la_log "          _la_gdb_prefix: $_la_gdb_prefix"
+_la_log "         _la_placeholder: $_la_placeholder"
+_la_log "  _la_target_wrapper_dir: $_la_target_wrapper_dir"
```

Comment:
Alternatively, you could just do this:
```suggestion
_la_log "  _la_target_wrapper_dir: $_la_wrapper_dir"
```